contracts/cw3-flex-multisig/src/contract.rs

```diff
@@ -559,7 +560,7 @@ mod tests {
 
         // 3. (Optional) Set the multisig as the group owner
         if multisig_as_group_admin {
-            let update_admin = Cw4HandleMsg::UpdateAdmin {
```

Comment:
Beware that this message still exists. Better to use the one froom the spec, I think.

Reply:
Good point